test-clients/src/main/java/com/hedera/services/bdd/spec/fees/FeeCalculator.java

```diff
@@ -134,7 +139,7 @@ public long forActivityBasedOpWithDetails(
 			AtomicReference<FeeObject> obs
 	) throws Throwable {
 		FeeData activityMetrics = metricsFor(txn, numPayerSigs, metricsCalculator);
-		return forOpWithDetails(op, SubType.DEFAULT, activityMetrics, obs);
+		return forOp(op, SubType.DEFAULT, activityMetrics);
```

Comment:
This should not be changed. See:
https://github.com/hashgraph/hedera-services/blob/09d57eb17b71caa7de399a9c2045a8a413a5ee74/test-clients/src/main/java/com/hedera/services/bdd/spec/fees/FeeCalculator.java#L128